openquakeplatform/openquakeplatform/svir/views.py

```diff
@@ -250,21 +250,31 @@ def export_variables_data(request):
     be exported
 
     :param request:
-        A "GET" :class:`django.http.HttpRequest` object containing the
-        following parameter:
+        A "GET" or "POST" :class:`django.http.HttpRequest` object containing
+        the following parameters:
             * 'sv_variables_ids': a string of comma-separated ids of social
                                   vulnerability variables
             * 'country_iso_codes': a string of comma-separated country iso
-                                   codes
-            * 'export_geometries': a boolean indicating if the geometries of
-                                   countries need to be exported too
+                                   codes (optional - default: all countries)
+            * 'export_geometries': 'True' or 'False', indicating if also the
+                                   geometries of countries have to be exported
+                                   (optional - default: 'False')
     """
-    if not request.GET.get('sv_variables_ids'):
+    is_get = request.method == 'GET'  # otherwise it's a post
+    if ((is_get and not request.GET.get('sv_variables_ids'))
+            or (not is_get and not request.POST.get('sv_variables_ids'))):
         msg = ('A list of comma-separated social vulnerability variable names'
                ' must be specified')
         response = HttpResponse(msg, status="400")
         return response
-    country_iso_codes = request.GET.get('country_iso_codes')
```

Comment:
Thanks, it also made possible to simplify some other lines of code, removing redundancy